includes/gateway/class-omise-payment-installment.php

```diff
@@ -105,16 +106,26 @@ public function getTotalAmount()
 		return WC()->cart->total;
 	}
 
+	/**
+	 * Get the total amount of an order in cents
+	 */
+	public function convertToCents($amount)
+	{
+			return intval(floatval($amount) * 100);
+	}
+
 	/**
 	 * @inheritdoc
 	 */
 	public function charge($order_id, $order)
 	{
-		$requestData = $this->get_charge_request($order_id, $order);
+		$token = isset( $_POST['omise_token'] ) ? wc_clean( $_POST['omise_token'] ) : '';
+		$source = isset( $_POST['omise_source'] ) ? wc_clean( $_POST['omise_source'] ) : '';
```

Comment:
Let's move this to `get_charge_request` and we don't have to pass these.

Reply:
Nice.